python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -146,6 +152,9 @@ async def generate_request(
         if self.to_create_loop:
             self.create_handle_loop()
 
+        while self.model_update_lock.locked():
+            await asyncio.sleep(0.1)
```

Comment:
```suggestion
            await asyncio.sleep(0)
```
Sleep 0.1 doesn't have any actual meaning. Essentially, it makes the current coroutine be moved to the end of the event loop, waiting for the next round of the loop to continue execution.